pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -706,6 +706,7 @@ type ConsoleSource struct {
 // BEGIN Inteface -----------------------------
 
 type Interface struct {
+	XMLName             xml.Name         `xml:"interface"`
```

Comment:
this isn't being used, is it ?

Reply:
This is used to generate interface XML:

https://github.com/kubevirt/kubevirt/blob/072aa67c2cd984f52e00f00608a2b134d826b13e/pkg/virt-launcher/virtwrap/manager.go#L334

otherwise it generates XML with capital letter `<Interface />` which is not recognizable by libvirt